ASWF/meetings/tsc/2020-09-28.md

```diff
@@ -0,0 +1,222 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+September 28, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [ ] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [ ] Mark Titchener (_TSC_) - Foundry
+  * [X] Carl Rand (_TSC_) - Weta Digital
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [ ] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Bernard Lefebvre - Autodesk
+  * [X] Nigel Hadley - Foundry
+  * [X] Simran Spiller
+
+Apologies:
+  * Kevin Wheatley
+  * Mark Titchener
+
+# **OCIO TSC Meeting Notes**
+
+* TAC update on GSoC. Focus for next year:
+    - Michael: Good feedback on results. Likely that ASWF will participate 
+      again next year and are hopful that all projects can get a project. 
+      Should start thinking about where to focus. CUDA, Metal renderer impl 
+      seems like a good place to try again on.
+    - Doug: Good to have volunteer for each project. Is Hacktoberfest 
+      happening again this year?
+    - Michael: Good question. I believe that was based on a tag search.
+
+* Issue #1117 -- Help Wanted:
+    - Doug: Added new label help-wanted. Others use this too. Can start adding 
+      this to high priority issues. This issue is to create OFX bindings for 
+      OCIO v2. High priority since it would increase test coverage with so many 
+      supporting apps. Could get more practitioners and developers to download 
+      and test those.
+    - Michael: OFX is supported by Nuke; Curious if it would create conflict 
+      with existing OCIO v1 binary in that env.
+
+* Open PRs: 
+    - 1147 (optimization):
+        - Doug: Updates config default version and optimization improvements. 
+          When create new config, it is now v2 by default instead of v1. Other 
+          thing in PR enhances optimization flags. Now ability to control types 
+          of optimization. Two new flags: one simplifies ops down to component 
+          parts; CDL without power becomes just matrix if possible, etc. 
+          Benefit is that can be combined with another matrix op. Another flag 
+          controls whether SSE log and power are used. Does much faster version 
+          of these functions but not 32-bit exact - 16-bit half precision. 
+          Useful to turn it on and off, and off for unit tests since we’re 
+          comparing with higher precision results.
+        - Bernard: Fast log is also used for baking.
```

Comment:
I think Bernard said the opposite.  Suggest replacing with:

Bernard: Fast log is not used for baking.

Reply:
Thanks, fixed.